week-3/A-array-find/exercise.js

```diff
@@ -1,13 +1,19 @@
-/* 
-  You are given an array of test scores.
-  Any result over 60 is a pass. 
-  Using .find() (and any other array methods necessary), find the lowest pass score
+/*
+  You are given an array of names.
+  Using .find(), we'd like to find the first name which starts with A and is longer than 7 letters.
 */
 
-var testScores = [90, 50, 100, 25, 81, 66, 80];
-var lowestPassScore = findLowestPassScore(testScores);
+// write your code here
 
-console.log(lowestPassScore);
+var names = ["Rakesh", "Alexandra", "Annam", "Mikey", "Karim", "Ahmed"];
+var longNameThatStartsWithA = names.find(findLongNameThatStartsWithA);
+
+function findLongNameThatStartsWithA(name) {
+  return name[0].includes("A") && name.length > 7;
```

Comment:
Good! One point about checking for the letter "A" though - if `name` is a string, you know that `name[0]` will be just one character - so it can't really "include" anything. Checking if that character is equal to something is good enough :)